python/lsst/rubintv/production/monitorPlotting.py

```diff
@@ -37,14 +38,25 @@ def plotExp(exp, figure, saveFilename):
         The figure to use for plotting.
     saveFilename : `str`
         The filename to save the image to.
+    scalingOption : `int`
+        The choice of colormap normalization.
     """
     data = quickSmooth(exp.image.array, 1)
-    vmin = np.percentile(data, 1)
-    vmax = np.percentile(data, 99)
 
+    cmap = cm.gray
     figure.clear()
     ax1 = figure.add_subplot(111)
-    im1 = ax1.imshow(data, cmap='gray', origin='lower', vmin=vmin, vmax=vmax)
+    match scalingOption:
+        case "default":
+            vmin = np.percentile(data, 1)
+            vmax = np.percentile(data, 99)
+            im1 = ax1.imshow(data, cmap=cmap, origin='lower', vmin=vmin, vmax=vmax)
+        case "CCS": # The CCS-style scaling
```

Comment:
Very picky flake8 error, sorry: need an extra space before the inline comment, but I'm trying to keep the package flake8-clean despite the lack of tests.

Reply:
Thanks for catching that! I really thought I had run scons...